CoreWebServer/src/au/gov/asd/tac/constellation/webserver/services/AddRecordStore.java

```diff
@@ -182,6 +183,7 @@ public void callService(final PluginParameters parameters, final InputStream in,
         }
 
         addToGraph(graphId, rs, completeWithSchema, arrange, resetView);
+        return new ServiceResponse(SC_OK, "Successful"); 
```

Comment:
So in here it would be `HttpURLConnection.HTTP_OK`.
What is the reason to show the message "Successful" given the HTTP code will inform this. Does the message appear in the response output?
Also if you are going to keep it then its work making the string a constant.

Reply:
@arcturus2 It's not really used afterwards, as I am processing only if it's not a 200. I just typed it thinking it's better than an empty string. What do you think?

Reply:
If its not used then how about creating another constructor like this?

```
public ServiceResponse(int errorCode){
        this.errorCode = errorCode;
        this.errorMessage = null;
}
```

Reply:
Agree, fixed.